examples/medplum-patient-intake-demo/src/pages/IntakeResponsePage.tsx

```diff
@@ -79,7 +79,11 @@ export function IntakeResponsePage(): JSX.Element {
 
   const questionnaireResponse = medplum.readResource('QuestionnaireResponse', responseId).read();
 
-  const omittedItems = ['agreement-to-pay-for-treatment-help', 'notice-of-privacy-practices-help'];
+  const omittedItems = [
```

Comment:
Why are these items omitted?

Reply:
To simplify the display of the form response. 

Here is the diff:

| with omit | without omit |
| --- | --- |
| [Screencast from 15-07-2024 11:07:45.webm](https://github.com/user-attachments/assets/f432f81a-825a-41f8-8611-edc40c9bb108) | [Screencast from 15-07-2024 11:08:29.webm](https://github.com/user-attachments/assets/1085c12e-1bd2-4dad-9f00-e2bf89899783) |

Reply:
Thank you!